Lib/logging/__init__.py

```diff
@@ -164,7 +164,7 @@ def addLevelName(level, levelName):
         _nameToLevel[levelName] = level
 
 if hasattr(sys, "_getframe"):
-    currentframe = lambda: sys._getframe(1)
+    currentframe = lambda: sys._getframe(3)
```

Comment:
I can understand that you have to add extra frames to allow for code in the `LoggerAdapter`, but what about code that doesn't use an adapter? What changed in 3.11 and later to cause the related issue?